jakarta.ls/src/main/resources/bean-validation.json

```diff
@@ -0,0 +1,13 @@
+{
+    "@Email string field containing an email address": {
+      "prefix": "beanvalidationemail",
+      "body": [
+        "@Email",
+        "\tprivate String emailAddress;"
+      ],
+      "description": "Bean Validation Constraint on Email string field",
```

Comment:
Perhaps this would make more sense: "Denotes an email address constraint, which validates a well-formed email address."